src/Event/Event.js

```diff
@@ -293,6 +294,7 @@ const Event = ({
             backgroundColor: event.color || DEFAULT_COLOR,
           },
           containerStyle,
+          event.style,
```

Comment:
I think we need to define these new props to event proptype.

Reply:
@hoangnm done